.github/workflows/deploy.yml

```diff
@@ -0,0 +1,36 @@
+name: "Deploy"
+
+on:
+  workflow_dispatch:
```

Comment:
I wrote this file thinking that we were going to do docker images the "old way".
Please confirm whether I should delete it.

Reply:
@shlevy ?